keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/eventemitter/SerialisationUtils.java

```diff
@@ -7,8 +7,6 @@
 import io.cloudtrust.keycloak.customevent.IdentifiedEvent;
```

Comment:
The modification in this file has nothing to do with migration to keycloak 18.0.0. Despite the fact that the updates seems to not change anything to the serialization. Did you test it ?

Reply:
Fully tested before changes, changes applied, then retested...

Reply:
Okay, so it is ok for me